Formula/p/python@3.13.rb

```diff
@@ -27,7 +27,7 @@ class PythonAT313 < Formula
   depends_on "xz"
 
   uses_from_macos "bzip2"
-  uses_from_macos "expat", since: :sequoia
+  uses_from_macos "expat"
```

Comment:
Still discussing best option here. Main issue is propagation of brew `expat` linkage across dependents. As seen by this PR, it has already impacted the `*-gdb` and `graph-tool` formulae.